README.md

```diff
@@ -177,6 +177,181 @@ ApolloConverterFactory apolloConverterFactory = new ApolloConverterFactory.Build
         .build();
 ```
 
+## Support For Cached Responses
+
+Apollo GraphQL client allows you to cache responses, making it suitable for use even while offline. The client can be configured with 3 levels of caching:
+
+ - **HTTP Response Cache**: For caching raw http responses. These responses are stored in files.
```

Comment:
I don't think "These responses are stored in files" is necessary as that is an implementation detail